src/transformers/cache_utils.py

```diff
@@ -28,6 +28,9 @@ class Cache(torch.nn.Module):
     Base, abstract class for all caches. The actual data structure is specific to each subclass.
     """
 
+    is_static = False
+    is_sliding = False
```

Comment:
(double-check whether this new attribute is correct, after rebasing. I think we've added new cache classes since your original commit. In the encoder-decoder cache, these attributes should be loaded from the decoder cache)

Reply:
yes, I ran tests for generation in llama and all compile-static tests in whisper. Rebased main and `fix-copies` propagated changes to some new models